lldb/docs/lldb-gdb-remote.txt

```diff
@@ -38,7 +38,43 @@ read packet: +
 read packet: $OK#9a
 send packet: +
 
+//----------------------------------------------------------------------
+// "QSupported"
+//
+// BRIEF
+//  Query the GDB remote server for features it supports
+//
+// PRIORITY TO IMPLEMENT
+//  Optional.
+//----------------------------------------------------------------------
 
+QSupported is a standard GDB Remote Serial Protocol packet, but
+there are several additions to the response that lldb can parse.
+An example exchange:
+
+send packet: qSupported:xmlRegisters=i386,arm,mips,arc;multiprocess+;fork-events+;vfork-events+
+
+read packet: qXfer:features:read+;PacketSize=20000;qEcho+;native-signals+;SupportedCompressions=lzfse,zlib-deflate,lz4,lzma;SupportedWatchpointTypes=aarch64-mask,aarch64-bas;
+
+In this example, three lldb extensions are shown:
+
+  PacketSize=20000
+    The base16 maximum packet size that the stub can handle.
```

Comment:
I was mostly confused by the lack of a space between base and 16. "base 16" is recognisable, "base16" sounds like a nightclub.I was mostly confused by the lack of a space between base and 16. "base 16" is recognisable, "base16" sounds like a nightclub.

Reply:
OK, I can see that.  I fixed this and a handful of other basenn uses in the file (probably all from me over the years).